src/apis/Chorus.js

```diff
@@ -14,8 +15,7 @@ module.exports = class ChorusAPI extends APIWrapper {
   }
 
   request (endpoint, queryParams = {}) {
-    return snekfetch.get(API_URL + endpoint)
-      .query(queryParams)
-      .then(r => r.body)
+    return fetch(API_URL + endpoint + `?${qs.stringify(queryParams)}`)
```

Comment:
why +

Reply:
I've already explained this. Readability.